src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Styles/ComboBox.xaml

```diff
@@ -0,0 +1,387 @@
+                                            x:Name="ChevronIcon"
+                                            Margin="0"
+                                            VerticalAlignment="Center"
+                                            FontSize="{StaticResource ComboBoxChevronSize}"
+                                            Foreground="{DynamicResource ComboBoxDropDownGlyphForeground}"
+                                            RenderTransformOrigin="0.5, 0.5"
+                                            Symbol="ChevronDown24">
+                                            <controls:SymbolIcon.RenderTransform>
+                                                <RotateTransform Angle="0" />
+                                            </controls:SymbolIcon.RenderTransform>
+                                        </controls:SymbolIcon>
+                                    </Grid>
+                                    <Grid
+                                        Grid.Column="0"
+                                        Grid.ColumnSpan="2"
+                                        Margin="0">
+                                        <ToggleButton
+                                            Name="ToggleButton"
+                                            HorizontalAlignment="Stretch"
+                                            VerticalAlignment="Stretch"
+                                            ClickMode="Press"
+                                            Focusable="False"
+                                            Foreground="{TemplateBinding Foreground}"
+                                            IsChecked="{Binding Path=IsDropDownOpen, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}"
+                                            Style="{StaticResource DefaultComboBoxToggleButtonStyle}" />
+                                    </Grid>
+                                    <Grid Grid.Column="0" Margin="{TemplateBinding Padding}">
+                                        <TextBox
+                                            x:Name="PART_EditableTextBox"
+                                            HorizontalAlignment="Stretch"
+                                            VerticalAlignment="Stretch"
+                                            FontSize="{TemplateBinding FontSize}"
+                                            Foreground="{TemplateBinding Foreground}"
+                                            IsReadOnly="{TemplateBinding IsReadOnly}"
+                                            Style="{StaticResource DefaultComboBoxTextBoxStyle}" />
+                                    </Grid>
+                                </Grid>
+                                <Popup
+                                    x:Name="Popup"
+                                    MinWidth="{TemplateBinding ActualWidth}"
+                                    VerticalAlignment="Center"
+                                    AllowsTransparency="True"
```

Comment:
AllowsTransparency be set to true may cause performance issues. After set AllowsTransparency, the popup windows have to copy the window's render content to GDI. See https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-updatelayeredwindow and https://docs.microsoft.com/en-us/previous-versions/windows/desktop/legacy/ms633557(v=vs.85)

https://github.com/dotnet/wpf/blob/446d4d008fe56097db66d96e2310662b23112da4/src/Microsoft.DotNet.Wpf/src/WpfGfx/core/hw/d3dswapchainwithswdc.cpp#L142-L189

Reply:
I tried my changes on a sample app, if we are using `AllowsTransparency = True`, then the combo box appearance will be like this - 
<img width="116" alt="image" src="https://github.com/dotnet/wpf/assets/85492722/e7889676-c5dd-45f0-860f-96b8f4c3ff30">

If we will set it to `False` or if we remove it then it is going to change the combo box appearance, it will look like this - 
<img width="121" alt="image" src="https://github.com/dotnet/wpf/assets/85492722/f19517b7-0ec5-4fc7-9b16-6cfafa608d4c">

Also I compared the performance, the value difference is less.